src/load.c

```diff
@@ -83,30 +84,29 @@ static void load_submit(gauge_t snum, gauge_t mnum, gauge_t lnum) {
     }
   }
 #endif
-  if (cores > 0) {
-    snum /= cores;
-    mnum /= cores;
-    lnum /= cores;
-  }
 
-  value_list_t vl = VALUE_LIST_INIT;
-  value_t values[] = {
-      {.gauge = snum},
-      {.gauge = mnum},
-      {.gauge = lnum},
+  metric_family_t fam = {
+      .name = "system.load",
+      .help = "System load average over a given duration",
+      .type = METRIC_TYPE_GAUGE,
   };
 
-  vl.values = values;
-  vl.values_len = STATIC_ARRAY_SIZE(values);
-
-  sstrncpy(vl.plugin, "load", sizeof(vl.plugin));
-  sstrncpy(vl.type, "load", sizeof(vl.type));
-
   if (cores > 0) {
-    sstrncpy(vl.type_instance, "relative", sizeof(vl.type_instance));
+    fam.name = "system.load.scaled";
```

Comment:
IMHO reporting differently named metric should be a user decision, not one done by plugin itself.

Reply:
This is a bit hidden by the diff, but it *is* a user decision: line 81 evaluates `report_relative_load` and only sets `cores` if it is true. `report_relative_load` represents a configuration option.

Reply:
Ah, that's why it was doing `cores > 0` which I was wondering a bit about... What machine has zero CPUs? :-)